wye/workshops/urls.py

```diff
@@ -10,4 +11,6 @@
         WorkshopUpdate.as_view(), name="workshop_update"),
     url(r'^(?P<action>[active,deactive]+)/(?P<pk>\d+)/$',
         WorkshopToggleActive.as_view(), name="workshop_toggle"),
+    url(r'^presenter/(?P<action>[opt\-in,opt\-out]+)/(?P<pk>\d+)/$',
```

Comment:
I was also planing that.  so final url will look like this.  
workshop/opt-in/1
workshop/opt-out/1